### PR TITLE
Adds a Karma Reminder line while shuttle in flight

### PR DESCRIPTION
**Pull request by SamHPurp**
Since the Karma buttons got removed from their own tab, the amount of Karma (anecdotally) given has dropped significantly.

This adds a simple line just after the shuttle departs, this gives the crew a few minutes to contemplate their gift, rather than get caught up in the mayhem of round-end. It appears as so:
image

:cl: Purpose2
add: Adds a subtle chat-window reminder for karma on Shuttle departure. Disable/enable via preferences tab.
del: Removes the ugly round-end karma popup.
/ :cl:

### CODE REVIEW

**Comment by tigercat2000** [comment]
I'll point out that all of the other karma reminders are all able to be disabled, this probably should be too
But give me like five minutes

**Comment by Aurorablade** [comment]
Yeah it needs to be given its own turn off option

**Comment by uraniummeltdown** [comment]
possibly add a link that opens up the Give Karma interface

**Comment by SamHPurp** [comment]
@Aurorablade There is, give karma already. 🤣

**Comment by Fox-McCloud** [comment]
This needs a toggle; I don't want to be annoyed with a message about spending my karma every round; I can toggle the others off--this should be no exception.

**Comment by SamHPurp** [comment]
I'll add in a toggle-able option soon.

The reason I didn't initially was that the reason most people have the other one off, is due to the popup.

People happy with the wording? Its a shortened version of the popup...

**Comment by Ralta** [comment]
Verbage is good, maybe change the comma to a semicolon for grammar's sake but it's good in my opinion! 👍

**Comment by SamHPurp** [comment]
Toggle added... I can't seem to test it on my own test server as I'm lacking a local SQL setup... I'm unsure of the SQL work that might need to be added for this. @Fox-McCloud @Allfd either of you chaps help?

**Comment by taukausanake** [comment]
I see no reason to have this let alone have it toggleable. It doesn't get in the way of anything, barely seems like a bother honestly. Because of such it doesn't seem needed unless you want to have karma points circulated more which also doesn't seem as important because then there's the idea you NEED to spend karma

**Comment by Fox-McCloud** [comment]
I would just tie this to `togglekarmareminder`; there really isn't much reason to have multiple karma-reminder preferences.

**Comment by SamHPurp** [comment]
@Fox-McCloud imo, I'd leave it un-toggleable.... The whole reason that people have the karma reminder turned off, is because its a pop up....

**Comment by taukausanake** [comment]
Thought does come up for some branches of Paradise that might have karma disabled from #7141. This would persist unless there was a way to disable it. Might want to take that into consideration

**Comment by Twinmold93** [comment]
This one is FAR less intrusive than the other reminders, for all consideration. Could perhaps just remove the karma pop-up at end-round altogether. End-round should look like going to the wrong site in terms of pop-up.

**Comment by Twinmold93** [comment]
Shouldn't, rather.

**Comment by SamHPurp** [comment]
The more I think about this. I think I'd rather remove the toggable option entirely. And just remove the end-round popup.

Turning off that sodding pop up is the first thing people do when they see that option. That should tell us something.

My proposed message is seriously un-intrusive.

**Comment by taukausanake** [comment]
I haven't turned it off. Sometimes I forget to give karma and feel it is important to pass it on to whoever was best if not outstanding that round

**Comment by deep-fried-faceplant** [comment]
>You have not yet spent your karma for the round; was there a player who was worthy of receiving your reward?

I'm no english major, but I thought the message could use an edit. Y/N?

You haven't given your karma point to anyone this round yet; was there anyone worthy of it?

File: code/__DEFINES/preferences.dm [hunk 1]
```diff
@@ -25,6 +25,7 @@
 #define MEMBER_PUBLIC	8192
 #define CHAT_NO_ADMINLOGS 16384
 #define DONATOR_PUBLIC	32768
+#define DISABLE_KARMA_TEXT_REMINDER	65536
```
Comment (Crazylemon64) [inline code/__DEFINES/preferences.dm]: 65536 = 1024 * 64 = 2**10 * 2**6 = 2**16
This is bit 17/32 (on the premise that BYOND uses 32-bit integers)

**Comment by Fox-McCloud** [comment]
Let's go ahead and tie this to `DISABLE_KARMA_REMINDER` instead. 

We already have a karma reminder toggle, so it makes sense for this to be handled under that one; additionally, we have a limited numbers of bitflags for prefs toggles, and I'd hate for the last one to be consumed on something like this.

**Comment by TullyBurnalot** [comment]
@SamHPurp

**Comment by SamHPurp** [comment]
@Fox-McCloud @TullyBurnalot I understand. However, doing so will basically mean nothing. As everyone and their grandmother has the karma reminder turned off.

The reason for it being turned off is the popup. Purely the popup. If we are to tie it to the same flag, can I just delete that awful un-styled HTML popup?

**Comment by TullyBurnalot** [comment]
@SamHPurp Acceptable course of action. Feel free to remove the popup and move the whole "karma reminder" thing to your proposed message. Lot less intrusive.

:+1:

**Comment by SamHPurp** [comment]
I'll do that this evening. This'll only basically effect new players, as everyone else has that flag disabled ;)

**Comment by deep-fried-faceplant** [comment]
I keep it on, it's a helpful reminder.

**Comment by SamHPurp** [comment]
@TullyBurnalot @Fox-McCloud assuming Travis plays ball, this should now be good to go.

**Pull request merged**